sdk/table/azure-table/azure/table/_table_service_client.py

```diff
@@ -0,0 +1,268 @@
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def set_service_properties(
+            self,
+            analytics_logging=None,  # type: Optional[Any]
+            hour_metrics=None,  # type: Optional[Any]
+            minute_metrics=None,  # type: Optional[Any]
+            cors=None,  # type: Optional[Any]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Sets properties for an account's Table service endpoint,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+       :param cors: Cross-origin resource sharing rules
+       :type cors: Any
+       :param minute_metrics: Minute level metrics
+       :type minute_metrics: Any
+       :param hour_metrics: Hour level metrics
+       :type hour_metrics: Any
+       :param analytics_logging: Properties for analytics
+       :type analytics_logging: Any
+       :keyword callable cls: A custom type or function that will be passed the direct response
+       :return: None
+       :rtype: None
+       :raises: ~azure.core.exceptions.HttpResponseError
+       """
+        props = TableServiceProperties(
+            logging=analytics_logging,
+            hour_metrics=hour_metrics,
+            minute_metrics=minute_metrics,
+            cors=cors
+        )
+        try:
+            return self._client.service.set_properties(props, **kwargs)  # type: ignore
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def create_table(
+            self,
+            table_name,  # type: str
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> TableClient
+        """Creates a new table under the given account.
+
+        :param table_name: The Table name.
+        :type table_name: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: TableClient, or the result of cls(response)
+        :rtype: ~azure.table.TableClient
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        table_properties = TableProperties(table_name=table_name, **kwargs)
+        self._client.table.create(table_properties)
+        table = self.get_table_client(table=table_name)
+        return table
+
+    @distributed_trace
+    def delete_table(
+            self,
+            table_name,  # type: str
+            request_id_parameter=None,  # type: Optional[str]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Creates a new table under the given account.
+
+        :param request_id_parameter: Request Id parameter
+        :type request_id_parameter: str
+        :param table_name: The Table name.
+        :type table_name: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: None
```

Comment:
should this be None? below it looks like we're returning something. is this just metadata?